tests/beaker_tests/tacacs_global/tacacs_global_provider_defaults.rb

```diff
@@ -144,15 +144,18 @@
 
   # @step [Step] Checks tacacs_global resource on agent using resource cmd.
   step 'TestStep :: Check tacacs_global resource presence on agent' do
-    # Expected exit_code is 0 since this is a puppet resource cmd.
-    # Flag is set to false to check for presence of RegExp pattern in stdout.
-    cmd_str = PUPPET_BINPATH + 'resource tacacs_global default'
-    on(agent, cmd_str) do
-      search_pattern_in_output(stdout, { 'enable' => 'false' },
-                               false, self, logger)
+    if platform != :ios_xr
```

Comment:
Should the entire teststep be in the ios_xr ``if`` block, since it is a nop for XR?  Otherwise it looks like the teststep was executed.